addons/zeus/functions/fnc_bi_moduleProjectile.sqf

```diff
@@ -133,8 +136,11 @@ if (_activated) then {
             //--- Create sound source
             _soundSource = if (_soundSourceClass != "") then {createSoundSource [_soundSourceClass,_pos,[],0]} else {objnull};
 
-            //--- Play radio warning
-            [] call _fnc_playRadio;
+            // Added by ACE_zeus to toggle ordnance radio message
+            if (GETMVAR(GVAR(radioOrdnance),true)) then {
```

Comment:
For future contributions, there's no need to use GETMVAR for settings. Settings variables are guaranteed to always be defined after preInit. 

Simply this would suffice:
```sqf
if (GVAR(radioOrdnance)) then {
```